internal/errors/errors.go

```diff
@@ -194,6 +194,27 @@ func IgnoreResponseErrors(ignore ...func(resp *http.Response, err error) bool) f
 	}
 }
 
+func IgnoreMetalResponseErrors(ignore ...func(resp *http.Response, err error) bool) func(resp *http.Response, err error) error {
+	return func(resp *http.Response, err error) error {
+		var r *http.Response
+		if resp != nil {
+			r = resp
+		}
+		mute := false
```

Comment:
This duplicates the `IgnoreHttpResponseErrors` function further down in this file